src/Components/Modals/ActivationKeyWizard.js

```diff
@@ -62,80 +66,117 @@ const CreateActivationKeyWizard = ({
   const { data: activationKeys } = useActivationKeys();
   const { addSuccessNotification, addErrorNotification } = useNotifications();
   const [name, setName] = useState('');
-  const [description, setDescription] = useState();
-  const [workload, setWorkload] = useState(workloadOptions[0]);
+  const [description, setDescription] = useState(
+    activationKey?.description || ''
+  );
   const [extendedReleaseProduct, setExtendedReleaseProduct] = useState('');
   const [extendedReleaseVersion, setExtendedReleaseVersion] = useState('');
   const [extendedReleaseRepositories, setExtendedReleaseRepositories] =
     useState([]);
-  const [role, setRole] = useState('');
-  const [sla, setSla] = useState('');
-  const [usage, setUsage] = useState('');
+  const [isSuccess, setIsSuccess] = useState(false);
+  // const [isMutationLoading, setIsMutationLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
```

Comment:
Polish: I think this state can be removed - it looks like it is set in a bunch of places but never read